podcasts/Onboarding/LoginCoordinator.swift

```diff
@@ -0,0 +1,105 @@
+import Foundation
+
+class LoginCoordinator: ObservableObject {
```

Comment:
```suggestion
class LoginCoordinator {
```

(maybe you're planning having states published here, if that's the case just ignore my comment)

Reply:
I'll need this once I do the social login logic but it's not needing right now. Removed: fe38ec0b37e48fbd089d5c78844d10507e2e2f89

Reply:
Removed: fe38ec0b37e48fbd089d5c78844d10507e2e2f89